app.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "sinatra/base"
+require "./conda_repo"
+require "builder"
+
+redis = Redis.new(url: ENV["REDIS_SERVER"], driver: :hiredis)
+CONDA_REPO = CondaRepo.new(redis)
+CONDA_REPO.update_packages
+
+class CondaAPI < Sinatra::Base
+  get "/" do
+    "Hello World! #{CONDA_REPO.package_names.length} \n"
+  end
+
+  get "/packages.json" do
```

Comment:
Do we really want `.json` at the end of these urls?

Reply:
Survey says. No!